pkg/common/event/dml_event.go

```diff
@@ -542,7 +542,7 @@ func (t *DMLEvent) AppendRow(raw *common.RawKVEntry,
 			return errors.Trace(err)
 		}
 		if skip {
-			log.Info("ignore DML by filter", zap.Any("tableInfo", t.TableInfo), zap.Any("raw", raw))
+			log.Debug("ignore DML by filter", zap.Any("tableInfo", t.TableInfo), zap.Any("raw", raw))
```

Comment:
If the `tableInfo` or `raw` parameter is too large, this line of log code will significantly impact performance.